posts/2024-03-10-testing_plan-toby_hocking/index.qmd

```diff
@@ -22,7 +22,7 @@ Current testing is limited to package checks run on CI:
 
 Current performance testing is informal. For example [before release](https://github.com/Rdatatable/data.table/pull/5515) devs run `test.data.table(memtest=TRUE)` to examine how much memory is used during tests. If too much memory is used, then that could result in a check failure on CRAN. Sometimes [git bisect](https://github.com/Rdatatable/data.table/issues/5424) is used to find the commit which caused a performance regression. But there are no systematic performance tests that are regularly done, even though performance is a major feature of `data.table`.
 
-The lack of a performance testing framework to run on CI results in performance regressions. Because an emphasis in `data.table` is on handling large data sets, fast and memory-efficient code is essential, and that is a primary reason why people use `data.table` for their data analyses. Consequently, when new changes introduce performance regressions (for example, increased computation time), users regularly file issues related to performance. As of October 2022, a search for the keyword performance yields 161 closed and 97 open issues and pull requests. Currently, there is no systematic performance testing framework in place for `data.table` developers, which unfortunately makes it easy to inadvertently introduce changes that adversely affect performance, and this is a barrier to accepting code contributions.
+The lack of a performance testing framework to run on CI often results in performance regressions. Because an emphasis in `data.table` is on handling large data sets, fast and memory-efficient code is essential, and that is a primary reason why people use `data.table` for their data analyses. Consequently, when new changes introduce performance regressions (for example, increased computation time), users regularly file issues related to performance. As of October 2022, a search for the keyword performance yields 161 closed and 97 open issues and pull requests. Currently, there is no systematic performance testing framework in place for `data.table` developers, which unfortunately makes it easy to inadvertently introduce changes that adversely affect performance, and this is a barrier to accepting code contributions.
```

Comment:
This is probably okay without the change I'm suggesting (although I think that alone is not just the source that would eventually sustain performance regressions), but at the first glance when reading this, I took 'CI results' as one term which obfuscated the sentence flow so I thought another word that chips in between might help readers like me